src/ProductConstructionService/ProductConstructionService.DependencyFlow/BatchedPullRequestUpdater.cs

```diff
@@ -57,4 +61,29 @@ protected override async Task<IReadOnlyList<MergePolicyDefinition>> GetMergePoli
         RepositoryBranch? repositoryBranch = await _context.RepositoryBranches.FindAsync(_id.Repository, _id.Branch);
         return repositoryBranch?.PolicyObject?.MergePolicies ?? [];
     }
+
+    public override async Task<bool> UpdateAssetsAsync(Guid subscriptionId, SubscriptionType type, int buildId, string sourceRepo, string sourceSha, List<Maestro.Contracts.Asset> assets)
```

Comment:
Can we have a quick call around this tomorrow? I am curious about a few things

Reply:
sure, whenever you want, even today if you have the time